analysis/topEFT/topeft.py

```diff
@@ -410,9 +420,12 @@ def process(self, events):
                             values = v[cut]
                             hout['invmass'].fill(eft_coeff=eft_coeffs_cut, eft_err_coeff=eft_w2_coeffs_cut, sample=histAxisName, channel=ch, invmass=values, weight=weights_flat, systematic=syst,appl=appl)
                         elif var == 'm3l': 
-                            if ((ch in channels2LSS) or (ch in channels2LOS) or (ch in ['eeeSSoffZ', 'mmmSSoffZ', 'eeeSSonZ' , 'mmmSSonZ']) or (ch in channels4L)): continue
```

Comment:
We're not currently calculating `m3l` or filling the corresponding histogram, so I think it might make sense to just entirely remove this `if` block for now. But, if you do want to leave it in, I guess I'm not quite sure why just the `channels4L` channel name was edited? (Since most of the other channel names in this list do not exist at this point either...)

Reply:
That was probably due to me fixing the `4L` vs `4l` typo, and picking a different convention to what is now in the master branch.

Reply:
Just double checked, and we are using `4l` everywhere now, but I don't mind removing `m3l` altogether.

Reply:
Right, the `4l` is now consistent. I'm just saying I'm not sure why we are making a point of making that one part of the line match the true name of the category when the rest of names of the categories in the same line e.g. `eeeSSoffZ` does not even exist (similar for `mmmSSoffZ`, `eeeSSonZ` etc...). Anyway, if you are ok with removing `m3l` all together, then that's what I'd probably vote for, since we're not even calculating that variable at this point (and we can always add it back in later if we want it).

Reply:
Ah, I see your point now! Ok, I'll remove it.